evm/generators.go

```diff
@@ -905,10 +910,11 @@ func AddCLI(sourceCode, structName string, noformat, includemain bool) (string,
 		if formattingErr != nil {
 			return code, formattingErr
 		}
+
 		code = string(generatedCode)
 	}
-
-	return code, nil
+	codeImportsFixed := strings.ReplaceAll(code, "github.com/quan8/go-ethereum", "github.com/ethereum/go-ethereum")
```

Comment:
Why is this necessary? This looks like a problem in your local development environment that you are pushing a fix for to all users.

Reply:
You are totally right, this is not necessary. It was introduced in early stage of development to make it work. Removing it.

Reply:
@zomglings: I've updated the code to remove this import. But it keeps importing the wrong ones: 

```go
"github.com/quan8/go-ethereum/common/math"
	"github.com/quan8/go-ethereum/crypto"

Reply:
Interesting. I wonder why that is. Could it be an LLM issue?

Reply:
Imports fixed! It was missing from appending to `t.Specs` struct 😄